apps/api/src/reservation/core/entities/inventory-item.entity.spec.ts

```diff
@@ -1,16 +1,27 @@
 import { Test } from '@nestjs/testing';
 import { DatabaseModule } from 'shared/database/database.module';
-import { ReservationItem } from './reservation-item.entity';
 import { ReservationOverlapException } from '../exceptions/reservation-overlap.exception';
 import { Reservation } from './reservation.entity';
 import { MikroOrmModule } from '@mikro-orm/nestjs';
 import { addMinutes, subMinutes } from 'date-fns';
 import { ConfigModule } from '@nestjs/config';
 import databaseConfig from 'config/database.config';
+import { InventoryItem } from './inventory-item.entity';
+import { faker } from '@faker-js/faker';
 
 const USER_ID = 1;
 
-describe('ReservationItem entity', () => {
+function createInventoryItem() {
```

Comment:
Can we move this to the bottom as this is a "private" function that is used only in this file?